packages/plop/scripts/postpublish.js

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env node
+
+import fs from "fs";
+import * as path from "path";
+import * as url from "url";
+
+const __dirname = path.dirname(url.fileURLToPath(import.meta.url));
+
+const exists = fs.existsSync("./README.md");
+
+if (exists) {
+  // Restore version of README previously there.
+  fs.copyFileSync(
+    path.join(__dirname, "./README.md"),
+    path.join(__dirname, "../README.md")
+  );
+
+  fs.unlinkSync(path.join(__dirname, "./README.md"));
+}
```

Comment:
I have confirmed that both this and `prepublish.js` work properly.

However, if you cancel a publish (say, thanks to OTP bugging you), it will not ever execute the `postpublish` and pollute your staging area with the modified `README.md` files. Just worth mentioning